src/Controls/src/Core/PlatformConfiguration/WindowsSpecific/Page.cs

```diff
@@ -9,35 +9,56 @@ namespace Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific
 {
 	using FormsElement = Maui.Controls.Page;
 
-	/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="Type[@FullName='Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific.Page']/Docs/*" />
+	/// <summary>
+	/// The page instance that Microsoft.Maui.Controls created on the Windows platform.
```

Comment:
This isn't really correct 

This is just an extension class for setting platform specifics

Reply:
@jsuarezruiz